tests/console/python3_new_version_check.pm

```diff
@@ -32,27 +32,36 @@ sub run {
         add_suseconnect_product('sle-module-python3');
     }
 
-    # Test new python3 versions if any
-    my @python_versions = split(/,/, get_var('PYTHON_VERSIONS', '3.10'));
-    my $sub_version;
-    foreach my $python_version (@python_versions) {
-        record_info("python$python_version is tested now");
-        $sub_version = substr($python_version, 2);
-        my @python31x_results = split "\n", script_output("zypper se python3$sub_version | awk -F \'|\' \'/python3$sub_version/ {gsub(\" \", \"\"); print \$2}\'");
-        if ((scalar(@python31x_results) == 0) || ($python31x_results[0] ne "python3$sub_version")) {
-            die("Python $python_version not found in the repositories\n\n");
+    # Test all avaiable new python3 versions if any
+    my $ret = zypper_call('se "python3[0-9]*"', exitcode => [0, 104]);
+    if ($ret == 104) {
+        if (is_sle('>=15-SP5')) {
+            record_soft_failure('bsc#1211386 - python3 new packages are not available yet');
+            return;
         }
-        else {
-            zypper_call("install python3$sub_version");
-            # Running classic testing algorithm 'man_or_boy'. More info at:
-            # https://rosettacode.org/wiki/Man_or_boy_test
-            assert_script_run("[ -f man_or_boy.py ] || curl -O " . data_url("console/man_or_boy.py") . " || true");
-            my $man_or_boy = script_output("python3.$sub_version man_or_boy.py");
-            if ($man_or_boy != -67) {
-                die("Execution of 'man_or_boy.py' not correct\n");
-            }
+        die('No new python3 packages available');
+    }
+    my @python3_versions = split(/\n/, script_output(qq[zypper se 'python3[0-9]*' | awk -F '|' '/python3[0-9]/ {gsub(" ", ""); print \$2}' | awk -F '-' '{print \$1}' | uniq]));
+    record_info("All available new python3 versions are: @python3_versions");
+    my $sub_version;
+    foreach my $python3_version (@python3_versions) {
+        record_info("$python3_version is tested now");
```

Comment:
Please move the longer sentence to second part, it's not visible anyway.
like "Tested python$python3_version", "$python3_version is tested now"
Not sure what will fit in the square.

Reply:
or just `"Testing $python3_version", "$python3_version is tested now"`